doc/faq.rst

```diff
@@ -67,6 +67,22 @@ to not be included as default messages.
 You can see the plugin you need to explicitly :ref:`load in the technical reference
 <user_guide/checkers/extensions:optional checkers>`.
 
+I want to use pylint on save in my IDE, how can I do that ?
+-----------------------------------------------------------
+
+Don't do it, pylint is not fast enough for that and never will be. pylint is best suited
```

Comment:
I don't use the VS Code plugin, but I just installed it and runs fine on an M1 mac. Not everyone has fast hardware, I know, but I wonder since more people will by with passing year, maybe we don't need to memorialize this discouragement? 

I'm wondering if even just suggesting disabling duplicate-code or running with -E or something is better than "here be dragons"?

Reply:
Hmm, wow, I think I meant 'on keystroke'  instead of 'on save' here.

Reply:
Fixed in https://github.com/pylint-dev/pylint/compare/14d653e3d34c24885cadacb86f36334e8f059c9d..1698065d6d661eb69eba9d174d96844fd417a31a